tests/test_mkfunding.py

```diff
@@ -137,12 +137,14 @@ def test_mkfunding_bad_cantaffordfee():
     assert response.stderr.__contains__(b'can\'t afford fee')
 
 
-def test_mkfunding_good():
+def test_mkfunding_good_noabort():
     response = subprocess_run(
         [EXECUTABLE,
          INPUT_TXID, INPUT_TXOUTPUT, INPUT_AMOUNT,
          FEERATE_PER_KW, INPUT_PRIVKEY,
          LOCAL_FUNDING_PRIVKEY, REMOTE_FUNDING_PRIVKEY])
+    # prior to bug fix for issue #5363,
+    # subprocess_run had a return code of -6 (abort)
```

Comment:
apologies if this is an obvious question, but how does this catch bug for pre-5363 fix?